pom.xml

```diff
@@ -55,15 +55,15 @@
         <maven.compiler.target>1.8</maven.compiler.target>
 
         <!-- Kafka and it's dependencies MUST reflect what the Kafka version uses -->
-        <version.kafka>0.10.2.0</version.kafka>
+        <version.kafka>0.11.0.0</version.kafka>
         <version.kafka.scala>2.11</version.kafka.scala>
         <version.curator>2.11.0</version.curator>
-        <version.zookeeper>3.4.8</version.zookeeper>
+        <version.zookeeper>3.4.10</version.zookeeper>
         <version.jackson>2.8.5</version.jackson>
-        <version.org.slf4j>1.7.21</version.org.slf4j>
+        <version.org.slf4j>1.7.25</version.org.slf4j>
         <version.log4j>1.2.17</version.log4j>
         <!-- check new release version at https://github.com/confluentinc/schema-registry/releases -->
-        <version.confluent.platform>3.2.0</version.confluent.platform>
+        <version.confluent.platform>3.2.2</version.confluent.platform>
```

Comment:
As per http://docs.confluent.io/3.2.2/release-notes.html, 3.2.2 uses Kafka 0.10.2.1-cp2. Given this CP version is only used for the dependency to the Avro converter, I reckon it doesn't make any difference, though?